packages/grafana-ui/src/components/Alert/Alert.mdx

```diff
@@ -6,12 +6,153 @@ import { Alert } from './Alert';
+   - Shows red color on background and uses the `Exclamation-triangle` icon
+     <br />
+
+## <a name="main-elements"/>Main elements
+
+- Icon Status: represent the importance of the message through the color and icon for each status.
+
+- Alert Copy/Title: should be explicit, clear and short that explain the most important part of the information.
+
+- Details: precise description for the notification, telling users what they need to do.
+
+- Button and text links: only persistent notifications should contain text links or buttons.
+
+- Close Icon: appears when the notification can be dismissed by the user.
+  <br />
+
+## <a name="usage"/>Usage
+
+<br />
+### <a name="alert-inline" />
+Inline Banner
+
+Inline Banners are task-oriented alerts that notify users of the status of an action or system. These alerts are contextual — appearing above or below main content areas — allowing users to address issues or to read messages quickly.
+
+#### Do's
+
+    - Use the appropriate status for the message (info/success/warning/error).
+    - Use plain, simple language for the messages to ensure users understand them and know how to address them.
+    - Focus on notifications if they are relevant to the users' current workflow and should be addressed immediately.
+
+#### Don'ts
+
+    - Do not use Primary calls to actions.
+    - Do not hide other content with inline notifications.
+    - Do not repeat the alert copy/title on the details message.
+
+#### When to use
+
+    - To inform at that time how to continue a task in which users must perform an action.
+    - To provide some options for users to take action.
+    - To explain in detail the reasons why something can’t be done by users.
+
+#### Placement
+
+    - Should be placed immediately above or below the related element.
 
 ```jsx
-<Alert title="Some very important message" severity="info" />
+<Alert title={title} severity={severity} buttonContent={buttonContent} onRemove={onRemove}>
+  <VerticalGroup>
+    <div>Child content that includes some alert details, like maybe what actually happened.</div>
+  </VerticalGroup>
+</Alert>
 ```
 
-<Props of={Alert} />
+### <a name="alert-toast"/>Toast
+
+Toast notifications are short messages that appear in the top-right corner of the page and are automatically hidden after a certain amount of time – they provide live notifications, without interrupting the user experience or requiring an action to be taken.
+
+#### Do's
+
+    - Use this component using `AppEvents` and `getAppEvents()`.
+    - Show the `Toast` publishing the `appEvent` with the proper type and payload needed.
+
+```jsx
+import { AppEvents } from '@grafana/data';
+import { getAppEvents } from '@grafana/runtime';
+
+const appEvents = getAppEvents();
+appEvents.publish({
+  type: AppEvents.alertSuccess.name,
+  payload: [text + ': ' + resp.status + ' (' + resp.statusText + ')'],
+});
+```
+
+#### Don'ts
+
+    - Do not use this component straightfordward.
```

Comment:
I wanted to say that `Toast` can't be used as usual, an event has to be published specifying the type and payload. How can I say that?

Reply:
do we expose the event externally, or is that just a grafana internal thing? 
- if we expose it externally: `Do not use this component directly. Instead, emit the <whatever> event to trigger it's appearance`.
- if we don't expose it externally and we don't want someone to use the component directly, feels like we shouldn't expose this story at all. think you change the name to <whatever>.story.internal.tsx and it will keep it when running locally but not when published externally

Reply:
We expose the event externally, we saw someone using it in a plugin: [`buttonPanel`](https://github.com/speakyourcode/grafana-button-panel/blob/master/src/buttonPanel.tsx)

Reply:
this still needs changing.